### ISSUE TITLE
Image properties - Input fields are not clickable ( react + material ui )

### ISSUE THREAD

**Issue by skaskevire** [xdan/jodit#879]
**Jodit Version:**  3.18.4

**Browser:**  Chrome 
**OS:**  Windows/Mac
**Is React App:**  True

I applied jodit editor into react + material ui setup into modal window and see that inputs inside image properties are not focusable and not clickable. Everything except this works correctly. 
My setup was the following:
 Dialog>
       //jodit editor placed here
     <textarea id="editor" name="editor" />
 /Dialog>

![2022-08-18 16 37 52 localhost bdfdf3f4a771](https://user-images.githubusercontent.com/8543287/185432007-699f6f5f-bc59-49df-b864-db88ff9386f4.png)

The same issue i see with Link popup ( all fields there are not clickable ).  I noticed that on version 3.4.x it works correctly, but Image properties has similar issue.

Is this known defect, did someone faced similar behavior before?

Thank you

**Comment by xdan** [comment]
You should check with the browser inspector what exactly is preventing focus on the elements. Perhaps this is some kind of your overlay

**Comment by skaskevire** [comment]
Thank you for your answer, after some research i found out that this is _mui_ Dialog functionality prevent focus from leaving the modal while open ( link editor and image editor are separate modals ) and there is a property to disable this behavior:
![image](https://user-images.githubusercontent.com/8543287/187032204-a53aabe9-8f92-4884-8b0e-56b1700b020b.png)
( screenshot was taken from https://mui.com/material-ui/api/modal/#props ).

My setup was the following:

 Dialog>
     <textarea id="editor" name="editor" />
 /Dialog>

To summarize i would like to say that this is not an issue with Jodit editor, but probably it will be a good idea to update documentation of image-editor and link-editor plugins and other plugins which are opening in new modals to warn users about such potential behavior

**Comment by xdan** [comment]
This is the Material UI `<Dialog>`/`<Modal>` **focus trap** (`enforceFocus`), same as #1119: when the editor is inside a MUI Dialog, MUI keeps focus within that dialog, but Jodit's *Image properties* dialog (and other popups) are appended to `document.body` **outside** the MUI Dialog — so MUI steals focus back and the input fields can't be clicked/typed.

**Fix:** add `disableEnforceFocus` to the MUI Dialog wrapping the editor:

```jsx
<Dialog open={open} disableEnforceFocus>
  <JoditEditor … />
</Dialog>
```

Jodit's dialogs/popups live outside the React tree on purpose (so they can overflow their container), which conflicts with MUI's default focus enforcement. Closing as a known MUI-integration fix; reopen if `disableEnforceFocus` doesn't help on the current 4.12.x.
